test/testvm.py

```diff
@@ -615,9 +633,8 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
             return subprocess.Popen(cmd)
 
         quoted = " ".join("'%s'" % x for x in cmd)
-        cmd = [ "/bin/sh", "-c", quoted + " | tee " + self.run_dir + "/qemu-$$.log" ]
-
-        return subprocess.Popen(cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+        cmd = [ "/bin/sh", "-c", quoted + " </dev/null >/dev/null" ]
```

Comment:
This could be improved.  We don't need the shell, and we could redirect qemu output directly to a file.